modules/ROOT/pages/index.adoc

```diff
@@ -21,6 +25,7 @@ Anypoint Studio uses your system's native browser to display elements of the Any
 ** Other supported runtime versions work fine with both Java 7 or 8.
 
 * Studio 6 is not compatible with Mule 4.
+* From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.
```

Comment:
I would not duplicate the info here... if you want to add something maybe I would add a reference to the previous paragraph.

Reply:
Good catch!
Thanks!